### PR TITLE
Update to version v0.4.8.

### PR DESCRIPTION
**Pull request by benpatterson**
@jzoldak @clytwynec 

re: TE-1136

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -31,16 +31,16 @@
 
 # General information about the project.
 project = u'bok-choy'
-copyright = u'2015, EdX'
+copyright = u'2016, EdX'
```
Comment (clytwynec) [inline docs/conf.py]: I don't really know enough about copyright ... are we actually supposed to change this each year?

Comment (benpatterson) [inline docs/conf.py]: Yeah, moreso for good hygiene as I understand it.

**Comment by jzoldak** [comment]
:+1:

**Comment by clytwynec** [comment]
:+1:

**Pull request merged**